README.md

```diff
@@ -23,8 +23,8 @@ housekeeping:
   schema-name: my_db
   # Connection details
   data-source:
-    # The package of your driver class
-    driver-class-name: com.mysql.cj.jdbc.Driver
+    # The name of your JDBC Driver class
+    driver-class-name: org.mariadb.jdbc.Driver
     # JDBC URL for your database
     url: jdbc:mysql://foo1baz123.us-east-1.rds.amazonaws.com:3306/${housekeeping.schema-name}
```

Comment:
To be consistent with our recommendation of using the MariaDB driver we should also update the example JDBC connection url to `jdbc:mariadb://foo1baz123.us-east-1.rds.amazonaws.com:3306/${housekeeping.schema-name}`

Reply:
But the url I used during testing was `jdbc:mysql`...

Reply:
Yeah, can you try it with `mariadb`? According to their documentation it can work with either but if you want to have both drivers on the classpath and use the DriverManager to figure out which jar to load then the URL has to be `mariadb` so I think it's the more recommended approach.

https://mariadb.com/kb/en/library/about-mariadb-connector-j/#using-drivermanager

We appear to be using the "legacy" way of doing it using the Driver class instead of having everything determined from the URL, not sure we want to change that now though.

Reply:
Using `jdbc:mariadb:<restoftheurl>` works 👍 

And I'm not sure, but it looks like it's a one-line change. I can look into that if you think it would be better.

Reply:
I've also changed the line with "For example, Housekeeping can be set up to use a MySQL schema:" to say MariaDB instead of MySQL. Is it clear enough?